docs/my-website/docs/completion/function_call.md

```diff
@@ -103,11 +103,25 @@ messages = [
     {"role": "user", "content": "What is the weather like in Boston?"}
 ]
 
-def get_current_weather(location):
-  if location == "Boston, MA":
-    return "The weather is 12F"
+def get_current_weather(location: str, unit: str):
+    """Get the current weather in a given location
 
-functions = litellm.utils.function_to_dict(get_current_weather)
+    Parameters
+    ----------
+    location : str
+        The city and state, e.g. San Francisco, CA
+    unit : str {'celsius', 'fahrenheit'}
+        Temperature unit
+
+    Returns
+    -------
+    str
+        a sentence indicating the weather
+    """
+    if location == "Boston, MA":
+        return "The weather is 12F"
+
+functions = [litellm.utils.function_to_dict(get_current_weather)]
 
 response = completion(model="gpt-3.5-turbo-0613", messages=messages, functions=functions)
```

Comment:
`functions` as a parameter is of type `list`. retyped the functions variable to `list` above